src/plugins/data/public/ui/query_editor/query_editor.tsx

```diff
@@ -288,8 +309,8 @@ export default class QueryEditorUI extends Component<Props, State> {
                   appName={this.services.appName}
                 />
               </EuiFlexItem>
-              {this.state.isDataSourcesVisible && (
-                <EuiFlexItem grow={false} className={`${className}__dataSourceWrapper`}>
+              {this.state.isDataSetsVisible && (
```

Comment:
should this be `isDataSetVisible`? I see some are plural some are singular, do we plan to allow multiple dataSets/dataSources on UI or just one?

Reply:
scared me. had to check. yeah i modified it for this because plural makes sense for me  like its the selector which can select multiple datasets. But tbh, I was on the fence too. I just wanted to updated in this file so its both plural.

Reply:
got it, it's minor but i wasn't sure if it's on purpose. if it's plural should it be `areDataSetsVisible` or `dataSetsVisible`?